cfgov/templates/wagtailadmin/js/table-block.js

```diff
@@ -344,12 +396,14 @@
                 ui.$hiddenField.val( JSON.stringify( {
                     data:                      this.handsonTable.instance.getData(),
                     column_widths:             this.getColumnWidths(),
+                    sortable_types:            this.getSortableTypes(),
                     first_row_is_table_header: ui.$hasRowHeaderCheckbox.prop( 'checked' ),
                     first_col_is_header:       ui.$hasColHeaderCheckbox.prop( 'checked' ),
                     is_full_width:             ui.$isFullWidthCheckbox.prop( 'checked' ),
                     is_striped:                ui.$isTableStripedCheckbox.prop( 'checked' ),
                     is_stacked:                ui.$isStackedOnMobileCheckbox.prop( 'checked' ),
-                    fixed_col_widths:          ui.$fixedWidthColsCheckbox.prop( 'checked' )
+                    fixed_col_widths:          ui.$fixedWidthColsCheckbox.prop( 'checked' ),
+                    is_sortable:               ui.$isSortableCheckbox.prop( 'checked' )
                 } ) );
```

Comment:
Should we reorganize this so that `fixed_col_widths` and `column_widths` are together and `is_sortable` and `sortable_types` are together, or is it better to keep the `this.<whatever>` stuff at the top?